v0.1/elements/electrodeContact.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/elements/electrodeContact.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SANDS definition of an electrode contact",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type", 
+    "@id",
+    "identifier",
+    "coordinate"
+    ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/ElectrodeContact"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier.",
+      "pattern": "sands/elements/electrodeContact/([A-Za-z0-9_-]+).json"
+    },
+    "identifier": {
+      "type": "string",
+      "description": "Identifier of the electrode contact within files."
+    },
+    "coordinate": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.schema.json#definitions/COORDINATE",
+      "description": "Central coordinate of the electrode contact."
+    },
+    "definedIn":{
+	  "type": "array",
+	  "minItems": 1,
+	  "items": {
+	    "oneOf": [
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json#definitions/CSCS_FILE"},
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE"}
+	    ],
+	  },
+      "description": "File in which the coordinate of this electrode contact were defined."
+    },
+    "relatedRecordings":{
```

Comment:
Note: Needs to be added to wiki.

Reply:
yes. I will do this after the PR is through. I can add then in parallel the link to the raw json-schema as well.

Reply:
Perfect! Was just thought as a reminder anyway :)